esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -1385,12 +1393,23 @@ bool CWsDfuEx::DFUDeleteFiles(IEspContext &context, IEspDFUArrayActionRequest &r
         userdesc->set(username, context.queryPassword(), context.querySessionToken(), context.querySignature());
     }
 
-    StringBuffer returnStr;
+    StringBuffer returnStr, auditStr = (",FileAccess,WsDfu,DELETED,");
     IArrayOf<IEspDFUActionInfo> actionResults;
 
     StringArray superFiles, failedFiles;
-    doDeleteSuperFiles(req.getLogicalFiles(), userdesc, superFiles, failedFiles, returnStr, actionResults, req.getRemoveFromSuperfiles(), req.getRemoveRecursively());
-    doDeleteSubFiles(req.getLogicalFiles(), userdesc, superFiles, failedFiles, returnStr, actionResults, req.getRemoveFromSuperfiles(), req.getRemoveRecursively());
+
+    SocketEndpoint ep;
+    ep.setLocalHost(0);
+    ep.getUrlStr(auditStr);
```

Comment:
This field (after 'DELETED,') should be the component name, e.g. "myesp"

Reply:
Ok, I change it.